packages/nextjs-template/components/DendronBreadCrumb.tsx

```diff
@@ -32,10 +32,12 @@ export function DendronBreadCrumb(props: DendronCommonProps) {
   const noteParents = noteIdPareents.map((noteId) => props.notes[noteId]);
 
   return (
+    // @ts-ignore
```

Comment:
by adding these, can we make the ts compiler stricter now in nextjs-template?

Reply:
we should - will take a pass at it